aws_lambda_powertools/utilities/batch/middlewares.py

```diff
@@ -54,3 +56,58 @@ def batch_processor(
         processor.process()
 
     return handler(event, context)
+
+
+@lambda_handler_decorator
+def sqs_batch_processor(
```

Comment:
Suggestion: Bring this middleware under `sqs.py` as it's closer to its implementation.